src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2727,7 +2727,7 @@ private string GenerateCertificateThumbprintParameter()
 
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-macos`
Tests for (error, warning, etc) action preference.action preference of Ignore cannot be set as a preference variable
```powershell
Expected an exception, with FullyQualifiedErrorId 'System.NotSupportedException,Microsoft.PowerShell.Commands.GetProcessCommand' to be thrown, but no exception was thrown.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1: line 40
40:             } | Should -Throw -ErrorId 'System.NotSupportedException,Microsoft.PowerShell.Commands.GetProcessCommand' -PassThru
```